src/lambda_codebase/account_processing/get_account_regions.py

```diff
@@ -16,6 +16,24 @@
 ADF_ROLE_NAME = os.getenv("ADF_ROLE_NAME")
 AWS_PARTITION = os.getenv("AWS_PARTITION")
 
+def get_default_regions_for_account(ec2_client):
+    default_regions = [
+    region["RegionName"]
+    for region in ec2_client.describe_regions(
```

Comment:
Could you extract this `describe_regions` call into a variable, on a separate line before the `default_regions = [` line to improve the readability?